src/__tests__/actions/reactions/bookmark.js

```diff
@@ -0,0 +1,78 @@
+import MockAdapter from 'axios-mock-adapter';
+import axios from '../../../utils/axiosInstance';
+import { bookmarkArticle, unbookmarkArticle } from '../../../actions/reactions/bookmark';
+import {
+  BOOKMARK,
+  BOOKMARK_ERROR,
+  BOOKMARK_ON_ALL,
+  UNBOOKMARK,
+  UNBOOKMARK_ERROR,
+  UNBOOKMARK_ON_ALL,
+} from '../../../actions/types';
+
+const axiosMock = new MockAdapter(axios, { delayResponse: 500 });
+const slug = 'testing-123';
+const id = '3';
+const articlePage = 'allArticles';
+
+describe('Bookmark action tests', () => {
+  const dispatch = jest.fn();
+
+  afterEach(() => {
+    dispatch.mockRestore();
+    axiosMock.reset();
+  });
```

Comment:
I thought this was handled in the test setup file already.

Reply:
No it is not handled in the setup file